recipe/meta.yaml

```diff
@@ -39,22 +38,12 @@ requirements:
     - libpng
     - libxcb                # [linux]
     - pixman
-    - xorg-libice  # [linux]
-    - xorg-libsm  # [linux]
-    - xorg-libx11  # [linux]
-    - xorg-libxext  # [linux]
-    - xorg-libxrender  # [linux]
-    - zlib
-  run:
-    - xorg-libice  # [linux]
-    - xorg-libsm  # [linux]
-    - xorg-libx11  # [linux]
-    - xorg-libxext  # [linux]
-    - xorg-libxrender  # [linux]
-    # hmaarrfk: 2022/08
-    # While the dependencies below should be captured
-    # by run_exports, run_exports doesn't necessary
-    # bring in compile time depenencies
+    - xorg-libice           # [linux]
+    - xorg-libsm            # [linux]
+    - xorg-libx11           # [linux]
+    - xorg-libxext          # [linux]
+    - xorg-libxrender       # [linux]
+    - xorg-xorgproto        # [linux]
     - zlib
```

Comment:
The changes actually remove `zlib` from the runtime requirements. Its somewhat annoying to start to pull in our whole "build time requirements" packages at runtime for space and simplicity considerations.

We should likely move to a replacement for `pkg-config` instead that don't require privately linked library at compile time to "find" the package.